cabby/geo/map_processing/map_structure.py

```diff
@@ -98,15 +99,23 @@ def create_S2Graph(self, level: int):
 
         # Get cellids for POI.
         self.poi['cellids'] = self.poi['geometry'].apply(lambda x: util.
-        cellid_from_point(
-            x, level) if isinstance(x, Point) else util.cellid_from_polygon(x, 
-            level))
+                                                         cellid_from_point(
+                                                             x, level) if 
+                                                             isinstance(x, 
+                                                             Point) else util.
+                                                             cellid_from_polygon
+                                                             (x,level))
```

Comment:
As before, write a function -- this is too complex as a lambda expression, which should only be one liners: https://google.github.io/styleguide/pyguide.html#2104-decision